src/json_parser.h

```diff
@@ -18,7 +18,9 @@ class JSONParser {
   JSONParser();
   ~JSONParser() {}
   bool Parse(const std::string& content);
-  std::optional<std::string> GetTopLevelField(const std::string& field);
+  // TODO(addaleax): Add support for non-ASCII encoded string fields.
```

Comment:
Does this TODO with anna's name make sense here ?

Reply:
Yea, it was her suggestion - https://github.com/nodejs/node/pull/47588#discussion_r1168543604, so I thought of using her name here